project/assets/test/ui/chat/cutscene-full.chat

```diff
@@ -43,14 +43,14 @@ s: ._.; Uhh I mean like, Richie's been helping me figure out the kitchen stuff.
 [sensei_answers_praise]
 s1: /._. Hmm. Yes! Pretty good. Your rhythm's ahhh... a little off though!
 s: My... my rhythm?
-s1: Yes!/ That food should be coming out like pap-/pap-/pap-/pap-/pap!/ Right now you're more like pappap,// pappa/ pap.
+s1: Yes! That food should be coming out like pap-|pap-|pap-|pap-|pap!| Right now you're more like pappap,|| pappa| pap.
```

Comment:
Restore these missing lull characters. They seem redundant, but the lull character generation does not add lull characters to dialog which already has lull characters.

Reply:
Restored missing lull character, `Yes!/ That food `